app/src/main/java/com/duckduckgo/app/browser/favicon/FaviconDownloader.kt

```diff
@@ -42,19 +49,14 @@ interface FaviconDownloader {
 class GlideFaviconDownloader @Inject constructor(
     private val context: Context,
     private val dispatcherProvider: DispatcherProvider,
+    private val androidBrowserConfigFeature: AndroidBrowserConfigFeature,
 ) : FaviconDownloader {
 
-    override suspend fun getFaviconFromDisk(file: File): Bitmap? {
-        return withContext(dispatcherProvider.io()) {
-            return@withContext runCatching {
-                Glide.with(context)
-                    .asBitmap()
-                    .load(file)
-                    .diskCacheStrategy(DiskCacheStrategy.NONE)
-                    .skipMemoryCache(true)
-                    .submit()
-                    .get()
-            }.getOrNull()
+    override suspend fun getFaviconFromDisk(file: File): Bitmap? = withContext(dispatcherProvider.io()) {
+        if (androidBrowserConfigFeature.glideSuspend().isEnabled()) {
```

Comment:
Since this flag is going to be accessed quite often, should we cache its value?

Reply:
I think it's fine for now.


Reminder to implement the [cache for the toggle stores](https://app.asana.com/1/137249556945/project/1202552961248957/task/1209674130594860?focus=true)